orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationModuleConfig.java

```diff
@@ -96,9 +95,9 @@ public class OrchestrationModuleConfig {
    */
   @Nullable GroundingModuleConfig groundingConfig;
 
-  @Nullable InputTranslationModuleConfig inputTranslationConfig;
+  @Nullable SAPDocumentTranslation inputTranslationConfig;
```

Comment:
Breaking changes for 3 orchestration classes?

Reply:
Then it looks good, no changes to the e2e tests. Also please add release notes just in case for "migration"